docs/openmetadata-apis/schemas/entities/accesscontrolrule.md

```diff
@@ -9,20 +9,46 @@ Type: `object`
 This schema <u>does not</u> accept additional properties.
 
 ## Properties
-- **name**
-  - Name that identifies this Rule.
+- **name** `required`
+  - Name for this Rule.
   - Type: `string`
-- **filters** `required`
-  - $ref: [filters.json#/definitions/filters](filters.md#filters)
-- **actions** `required`
-  - A set of access control enforcements to take on the entities.
-  - Type: `array`
-  - Item Count:  &ge; 1
-    - **Items**
+- **entityTypeAttr**
+  - Entity type that the rule should match on.
+  - Type: `string`
+- **entityTagAttr**
+  - Entity tag that the rule should match on.
+  - $ref: [../../type/tagLabel.json#/definitions/tagFQN](../types/taglabel.md#tagfqn)
+- **userRoleAttr**
+  - Role of the user that the rule should match on.
+  - $ref: [../teams/team.json#/definitions/teamName](team.md#teamname)
+- **operation**
+  - Operation on the entity.
+  - $ref: [#/definitions/operation](#operation)
+- **allow**
+  - Allow or Deny operation on the entity.
+  - Type: `boolean`
+  - Default: _false_
+- **priority**
+  - Priority of this rule among all rules across all policies.
+  - Type: `integer`
+  - Default: `250000`
 - **enabled**
   - Is the rule enabled.
   - Type: `boolean`
   - Default: _true_
 
 
-_This document was updated on: Tuesday, December 14, 2021_
\ No newline at end of file
+## Type definitions in this schema
+### operation
+
+- This schema defines all possible operations on metadata of data entities.
+- Type: `string`
+- The value is restricted to the following: 
+  1. _"SuggestDescription"_
+  2. _"SuggestTags"_
+  3. _"UpdateDescription"_
+  4. _"UpdateOwner"_
+  5. _"UpdateTags"_
```

Comment:
@parthp2107 Can you rebase off of main and regenerate the docs, - we just added a new operation with #2406

Reply:
Updated in the latest commit